fastlane/Fastfile

```diff
@@ -39,6 +39,34 @@ platform :android do
     attach_changelog_to_main(options)
   end
 
+  desc "Tag release version with latest if necessary"
+  lane :tag_release_with_latest_if_needed do |options|
+    release_version = options[:release_version]
+
+    lines_in_changelog = File.open("../CHANGELOG.md").to_a
+    first_line = lines_in_changelog.first
+    latest_version = first_line.split("## ").last.chomp
+    puts "Latest version is #{latest_version}"
```

Comment:
less efficient but maybe more reliable: checkout the current latest tag and read the version from library.gradle?

Reply:
That might be more reliable yes. We actually don't make sure what we add to the CHANGELOG is in order (which I realized when I was opening the PR) so it might make more sense to do it that way. I will try that out.

Reply:
Updated!